packages/web/src/features/landing/components/ServiceCard.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+
+import React from "react";
+import styled from "styled-components";
+import Image from "next/image";
+
+import Card from "@sparcs-clubs/web/common/components/Card";
+
+import arrowRightSvg from "@sparcs-clubs/web/assets/arrow-right.svg";
+
+interface ServiceCardProps {
+  serviceTitle: string;
+  serviceLink: string;
+}
+
+const ServiceCardInner = styled(Card)`
+  gap: 16px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 45px;
```

Comment:
최대한 피그마랑 똑같이 해주세요! 자세히 보시면 해당 컴포넌트는 피그마에서 height가 45px로 fixed가 아니라 "fill"일거에요
즉 `flex: 1`와 같이 구현하셔야 합니다!!

```suggestion
  flex: 1;
  padding: 12px 20px;
```